transaction/delivery.go

```diff
@@ -116,6 +116,10 @@ func (d *Delivery) applyFees(f *fees) *Delivery {
 	return &dd
 }
 
+func (t Delivery) Hash() string {
+	return fmt.Sprintf("%s:%d:%d", t.Sender, t.ChainID, t.Nonce)
```

Comment:
shouldn't it be named something else? like ID() or something, its not hashing anything